opus/application/static_media/js/opus.js

```diff
@@ -589,6 +596,10 @@ var opus = {
  *
  */
 $(document).ready(function() {
+    // just for browse and cart, as we need to abstract some functions easier
+    // needs to be done in the document.ready because o_cart & o_browse need to be defined first
+    opus.namespaceDict = { "cart": o_cart, "browse": o_browse };
```

Comment:
If this maps "view" to namespace, I think it should have a name that reflects that. Like viewToNamespaceDict.